config/default.yml

```diff
@@ -553,6 +553,10 @@ Style/TrivialAccessors:
     - to_s
     - to_sym
 
+Style/UnneededPercentX:
+  # Enforce using %x when the command string spans multiple lines.
+  EnforcePercentXForMultiLine: false
```

Comment:
Exactly. Maybe the cop could be renamed to `CommandLiteral` (inspired by `RegexpLiteral`)?

Then we keep enforcing backticks when a one-line literal doesn’t contain “inner backticks”, and enforce (or allow) `%w` around multi-line command literals.

Or should the configuration rather be `PreferredStyleForMultiline` with values `percent_x` and `backtick`, so that one of the styles is always enforced?